nixos/modules/services/networking/cjdns.nix

```diff
@@ -241,10 +258,13 @@ in
                 | ${pkg}/bin/cjdroute
          ''
       );
-
+      
+      onFailure = ["cjdns-restart.service"];
```

Comment:
From understanding this is the job of `Restart = "always";`, which does the same, right?